### ISSUE TITLE
ZFS not in path rancher-agent and rancherOS

### ISSUE THREAD

**Issue by segator** [rancher/rke#979]
**RKE version:**
version that comes with rancher/rancher server
**Docker version: (`docker version`,`docker info` preferred)**

**Operating system and kernel: (`cat /etc/os-release`, `uname -r` preferred)**
rancherOS latest
**Type/provider of hosts: (VirtualBox/Bare-metal/AWS/GCE/DO)**
vmware/scaleway

Created a custom cluster and execute the docker provided by rancher server.

if I check the logs of rancher-agent container I see this errors
1021 10:16:30.000462   17812 fs.go:425] Stat fs failed. Error: exec: "zfs": executable file not found in $PATH: "zfs zfs get -Hp all zpool/docker" =>
E1021 10:16:30.994351   17812 watcher.go:98] encountered error getting zfs filesystem: zpool/docker: exec: "zfs": executable file not found in $PATH: "zfs zfs get -Hp all zpool/docker" =>
E1021 10:16:30.994388   17812 watcher.go:63] encountered error refreshing zfs watcher: exec: "zfs": executable file not found in $PATH: "zfs zfs get -Hp all zpool/docker" =

Docker server is installed using ZFS Storage zpool/docker is the zpool mapped using rancherOS ZFS

**Comment by pdf** [comment]
See also: rancher/hyperkube#11 though this needs to be applied to multiple branches.

**Comment by smedefind** [comment]
Will this ever be marked for a release?

We're paying for Rancher and not being able use ZFS for kubelet is hampering our cluster.

**Comment by alena1108** [comment]
https://github.com/kubernetes/kubernetes/issues/54765

**Comment by alena1108** [comment]
Can be tested through Rancher `rancher/rancher:v2.2.5-rc3`. Docker ssh to any of the k8s system containers and check if zfs-utils tool is present.

**Comment by jiaqiluo** [comment]
The bug fix is validated in Rancher:v2.2.5

Steps:
- run rancher: v2.2.5 container
- add a single-node custom cluster using the default k8s version: v1.1.4.3-rancher1-1
- after the cluster is active, ssh into the node 
- `docker ps | grep hyperkube` to find containers using the `rancher/hyperkube` image
```
$ docker ps | grep hyperkube
b6f1a7d7f717        rancher/hyperkube:v1.14.3-rancher1                                                                                        "/opt/rke-tools/en..."   35 minutes ago      Up 35 minutes                           kube-proxy
2765534cf235        rancher/hyperkube:v1.14.3-rancher1                                                                                        "/opt/rke-tools/en..."   35 minutes ago      Up 35 minutes                           kubelet
53dc8b45fae5        rancher/hyperkube:v1.14.3-rancher1                                                                                        "/opt/rke-tools/en..."   35 minutes ago      Up 35 minutes                           kube-scheduler
5042e995d824        rancher/hyperkube:v1.14.3-rancher1                                                                                        "/opt/rke-tools/en..."   35 minutes ago      Up 35 minutes                           kube-controller-manager
059d24ff6601        rancher/hyperkube:v1.14.3-rancher1                                                                                        "/opt/rke-tools/en..."   35 minutes ago      Up 35 minutes                           kube-apiserver
```
- `docker exec` into one of them, in my case it is `kubelet` 
-  check that zfs tools exist
```
# which zpool
/sbin/zpool
# which zfs
/sbin/zfs
```
